measurement/dns/answer.go

```diff
@@ -27,7 +27,7 @@ import (
 // First two records from the response decoded by the probe, if they are
 // TXT or SOA; other RR can be decoded from "abuf"
 type Answer struct {
-    Data struct {
+    data struct {
```

Comment:
That's going to leave it in an ugly mix where some objects have accessors and other haven't, as said, Go json has limits and I need to intercept unmarshaling because json objects with the same label can be different objects depending on measurement type (this is RIPE Atlas API, nothing I can do anything about) and I can't unmarshal the object from the object (loops and dies).